app/src/main/res/values/dimens.xml

```diff
@@ -6,6 +6,20 @@
     <dimen name="splash_title_textsize">36sp</dimen>
     <dimen name="splash_subtitle_textsize">20sp</dimen>
 
+    <dimen name="edit_text_textsize">24sp</dimen>
+    <dimen name="edit_text_linespacingextra">3sp</dimen>
+
+    <dimen name="text_view_label_textsize">16sp</dimen>
+    <dimen name="text_view_label_marginbottom">5dp</dimen>
+
+    <dimen name="button_textsize">14sp</dimen>
+    <dimen name="button_paddingtop">13dp</dimen>
```

Comment:
These can all probably be 16dp, or at the least 14 dp. (14 vs 13 dp is barely visible)